src/quo/components/community/channel_action/style.cljs

```diff
@@ -0,0 +1,27 @@
+(ns quo.components.community.channel-action.style
+  (:require
+    [quo.foundations.colors :as colors]))
+
+(defn channel-action-container
+  [{:keys [big? disabled?]}]
+  (cond-> {}
+    disabled?  (assoc :opacity 0.3)
+    ;; NOTE: The big action has to fill the available space when there's a small
+    ;; action next to it. If alone, take an approx. proportion of space ~2/3.
+    ;; Using fixed sizes according to the designs will make it look wrong on
+    ;; all other devices.
+    big?       (assoc :flex-grow 1)
```

Comment:
Hi, Thank you for addressing this. Do we need this comment? I think code is self explanatory.